### PR TITLE
HDE interface module to send human data to simulink through a custom device driver

### PR DESCRIPTION
**Pull request by Yeshasvitvs**

### CODE REVIEW

File: hde-interface/src/HDEInterfaceModule.cpp [hunk 1]
```diff
@@ -234,6 +238,13 @@ bool HDEInterfaceModule::updateModule()
         yError() << "HDEInterfaceModule: DoFs mismatch between config file and human joint dynamics";
         return false;
     }
+
+    for(int j=1; j <= hde_controlboard_driver_ptr->number_of_dofs; j++)
+    {
+        hde_controlboard_driver_ptr->joint_positions[j] = hde_controlboard_driver_ptr->joint_positions_rad[j]*(180/M_PI);
+        hde_controlboard_driver_ptr->joint_velocities[j] = hde_controlboard_driver_ptr->joint_velocities_rad[j]*(180/M_PI);
+        hde_controlboard_driver_ptr->joint_accelerations[j] = hde_controlboard_driver_ptr->joint_accelerations_rad[j]*(180/M_PI);
```
Comment (traversaro) [inline hde-interface/src/HDEInterfaceModule.cpp]: `M_PI` is not a standard macro, after the demo remember to make sure that this code compiles under Windows.

Comment (Yeshasvitvs) [reply hde-interface/src/HDEInterfaceModule.cpp]: @traversaro Ok :+1:

**Comment by Yeshasvitvs** [comment]
[HumanControlBoard](https://github.com/Yeshasvitvs/human-dynamics-estimation/tree/feature/humanstateprovider_multi_thread_ik/devices/HumanControlBoard) of HDEv2 does the same job of wrapping the human data from HDEv2 devices and send the data as a motor control board device. So, this PR is not relevant anymore. Closing it. 

@traversaro @diegoferigo

**Comment by diegoferigo** [comment]
:top: Well done @Yeshasvitvs @lrapetti

**Pull request closed without merging**